cf/functions.py

```diff
@@ -3096,6 +3096,43 @@ def default_netCDF_fillvals():
     return netCDF4.default_fillvals
 
 
+def size(a):
+    """Return the number of elements.
```

Comment:
+1 for documenting this by hand and not going with the blanket "This docstring was copied from numpy.size. Some inconsistencies with the cf version may exist." :smiley: